crl-janitor/pubsub.tf

```diff
@@ -0,0 +1,44 @@
+resource "google_pubsub_topic" "crl-janitor-pubsub-topic" {
+  count = var.enable ? 1 : 0
+
+
+  project = var.google_project
+  name = "${local.owner}-${local.service}-pubsub-topic"
+}
+
+resource "google_pubsub_subscription" "crl-janitor-pubsub-subscription" {
+  count = var.enable ? 1 : 0
+
+
+  project = var.google_project
+  name = "${local.owner}-${local.service}-pubsub-sub"
```

Comment:
I think the general convention we've been following for the order of the namespacing is service-owner, not owner-service. For the SAs for example. Am open to changing that but it should be a deliberate change.

Reply:
done